notifications.py

```diff
@@ -0,0 +1,16 @@
+def median_list(nums):
+    nums = sorted(nums)
+    if len(nums) % 2 == 1:
+        return nums[len(nums)//2]

```

Comment:
I think there should be a space before and after '//' for consistency with other binary operator.

The best way forward to make sure you're following a consistent coding style is to run pylint on your files to check for formatting inconsistencies.

Reply:
Got you: Thanks for letting me know maintain consistency is also important. I have updated the code:
if (len(nums) % 2) == 1:
        return nums[len(nums) // 2]
    else:
        return int(((nums[len(nums) // 2) + 1] + nums[( len(nums) // 2) - 1]) / 2.0)

Reply:
The code wasn't updated to reflect the new style. Please update the pull request.

Also len(nums) // 2 is repeated 3 times. I should be put into a variable.